app/controllers/relationships_controller.rb

```diff
@@ -0,0 +1,21 @@
+class RelationshipsController < ApplicationController
+  before_action :logged_in_user
+
+  def create
+    user = User.find_by(id: params[:followed_id])
+    current_user.follow(user)
+    respond_to do |format|
```

Comment:
giải thích chị line code này nào

Reply:
dạ, hàm create dùng để tạo follow, tức là khi ta ấn vào nút follow trên màn hình thì hàm này sẽ được gọi, line 5 dùng để tìm id của người mình muốn follow, line 6 để lưu nhớ follow, line 7 là để xuất ra ngoài html và cả thêm file js nữa ạ

Reply:
`  respond_to do |format|` đang hỏi cái này em naf?

Reply:
Dạ cái này là  web-service, theo e hiểu la khi respond to do |format| thì nó sẽ gửi theo kiểu mình yêu cầu, như ở trên thì nó gửi tới file html, và file js khi được gọi ạ